nss/src/cache/mod.rs

```diff
@@ -309,6 +321,7 @@ impl CacheDBBuilder {
         if shadow_mode >= ShadowMode::ReadOnly {
             let shadow_db = shadow_db.to_str().unwrap();
 
+            debug!("attaching shadow database");
```

Comment:
It's not a good idea to have this type of information printed on the system logs. Would you mind removing this?

Reply:
It's removed in the latest commit, I used this for debugging purposes.